src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,32 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+    @Test
+    void splitString() {
+        String [] result = "1,2".split(",");
+        Assertions.assertThat(result).containsExactly("1","2");
```

Comment:
```suggestion
        Assertions.assertThat(result).containsExactly("1", "2");
```
소소하지만 이런 포맷팅도 생활화하시면 좋을 것 같습니다ㅎㅎ